nrc_ws/src/nrc_nav/src/nrc_drive_dr.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python
+
+import rospy
+
+import time
+import csv
+from numpy import genfromtxt
+
+from nrc_msgs.msg import DriveCommand
+
+# Global robot object
+command_pub = None
+# global time
+start_time = time.time()
+# time index for trajectory gen instructions
+instruction_index = 0
+# instructions pulled in at the start of code running
+instructions = None
+
+
+def generate_drive_command(timer_event):
+    # make sure we can access the global time
+    global start_time
+    # get local time at every tick to compare to the path-gen (in seconds)
+    local_time = time.ctime(start_time)
```

Comment:
This converts the time into a timestamp as far as I can tell. The way I normally approach this is to do `local time = time.time() - start_time)`